### ISSUE TITLE
AlexNet Filters and Learning Rate

### ISSUE THREAD

**Issue by FelipeMarcelino** [deep-ufmg/praticas#3]
No notebook 02.1 - [Estrategias-de-treino.ipynb,](https://github.com/deep-ufmg/praticas/blob/master/03-Modulo/02.1%20-%20Estrategias-de-treino.ipynb) do módulo 03, a primeira implementação usando classes da  rede AlexNet não deveria ter 96 filtros ao invés de 64 na primeira Conv2d?

**Comment by FelipeMarcelino** [comment]
No notebook 02.1 - [Estrategias-de-treino.ipynb,](https://github.com/deep-ufmg/praticas/blob/master/03-Modulo/02.1%20-%20Estrategias-de-treino.ipynb) do módulo 03, o learning rate do SGD não deveria ser 0.01 invés de 0.001, pois usando este último valor no pytorch, a rede demora a convergir!
